src/main/java/com/artistsolutions/betting/controllers/TeamController.java

```diff
@@ -1,29 +1,45 @@
 package com.artistsolutions.betting.controllers;
-
-import com.artistsolutions.betting.dto.Team;
+import com.artistsolutions.betting.dto.TeamDTO;
+import com.artistsolutions.betting.entity.Team;
+import com.artistsolutions.betting.repository.TeamRepository;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.*;
+import java.util.List;
+import java.util.stream.Collectors;
 
 @RestController
 @RequestMapping("/teams")
 public class TeamController {
 
+    @Autowired private TeamRepository teamRepository;
+    @Autowired private ModelMapper modelMapper;
+
     @GetMapping()
-    public void getTeams() {
-        System.out.println("All teams returned");
+    public List<TeamDTO> getTeams() {
+        List<Team> teams = teamRepository.findAll();
+        return teams.stream().map(team -> modelMapper.map(team,TeamDTO.class)).collect(Collectors.toList());
```

Comment:
This probably can be done easier, not to map each element one by one. We will see together. 
By the way, this is lambda expression :)